BenchmarkVCFs/BenchmarkVCFs.wdl

```diff
@@ -31,6 +31,7 @@ workflow Benchmark {
         Boolean passingOnly=true
         String? vcfScoreField
         String? dummyInputForTerraCallCaching
+        Boolean fingerprint_toggle = true
```

Comment:
I would name this something where it's clear what true vs false means, e.g. `run_fingerprint = true` means to run the fingerprint task.

Reply:
Changed it to runFingerprintCheck - remembered the camel case!

Reply:
For variables in WDL it's also "ok" to use snake_case, but this works too. The last time I mentioned using CapitalCamelCase for tasks/workflow names since those are "big" object blocks.